From 506891679646928b85fdac5eac3cadc95c4145ff Mon Sep 17 00:00:00 2001
From: Vibhore Vardhan <vibhore@ti.com>
Date: Wed, 5 Feb 2025 14:28:00 -0600
Subject: [PATCH 09/68] PENDING: arm64: dts: ti: k3-am62a7-sk: Add alias for
 RTC

Adds alias for SoC RTC so that it gets assigned rtc0. PMIC node is
assisgned rtc1 so that PMIC RTC gets probed as rtc1. This makes it
consistent for testing with other AM62 devices where rtc0 is SoC
RTC.

Signed-off-by: Vibhore Vardhan <vibhore@ti.com>
Signed-off-by: Kendall Willis <k-willis@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am62a7-sk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
index c4be6bee5fe7..97509002303f 100644
--- a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
+++ b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
@@ -22,6 +22,8 @@ aliases {
 		serial3 = &main_uart1;
 		mmc0 = &sdhci0;
 		mmc1 = &sdhci1;
+		rtc0 = &wkup_rtc0;
+		rtc1 = &tps659312;
 	};
 
 	chosen {
-- 
2.47.2

